config/_default/menus/main.en.yaml

```
@@ -4391,16 +4391,11 @@ menu:
       identifier: cd_deployments_ciproviders
       parent: cd_deployments
       weight: 102
-    - name: Search and Manage
-      url: continuous_delivery/search
-      identifier: cd_search
-      parent: cd
-      weight: 2
     - name: Explorer
```

The page's new title is now "Explore CD Visibility Deployments" but that might a lot for the left nav menu. Shorter option:
```suggestion
    - name: Explore Deployments
```